src/tests/unit/tests/DetailsView/components/tab-stops/tab-stops-requirements-table.test.tsx

```diff
@@ -64,13 +65,13 @@ describe('TabStopsRequirementsTable', () => {
     test('renders result column', () => {
         const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
         const columns = testSubject.find(DetailsList).props().columns;
-        const tabStopsChoiceGroup = columns[1].onRender(requirementsList[0]);
+        const tabStopsChoiceGroup = columns[1].onRender(requirementsList(false)[0]);
         expect(tabStopsChoiceGroup).toMatchSnapshot();
     });
 
     test('result column handlers', () => {
         const eventStub = new EventStubFactory().createKeypressEvent() as SupportedMouseEvent;
-        const actualRequirement = requirementsList[0]; // must match with state from builder which uses actual requirement.
+        const actualRequirement = requirementsList(false)[0]; // must match with state from builder which uses actual requirement.
```

Comment:
Should we have a test for when the feature flag is on, to verify the requirements text?